### PR TITLE
Feat: Added Cart page

### PR DESCRIPTION
**Pull request by kjais1720**
- Added cart page
Preview link - 
https://deploy-preview-4--crickstock.netlify.app/pages/cart.html

https://user-images.githubusercontent.com/66024105/154131082-2873d8b9-1bca-4aa6-a62a-f3743d8631e4.mp4

### CODE REVIEW

File: pages/cart.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <div class="d-flex f-wrap justify-c-center align-i-start gap-xlg pd-y-lg">
+            <div id="productCardsContainer" class="cart-items flex-col gap-md">
+                <!-- Cart Items to be added dynamically  -->
+            </div>
+            <aside class="cart-summary tr-card flex-col gap-md pd-md">
+                <h3 class="txt-semibold">Price details</h3>
+                <hr>
+                <div class="flex-col">
+                    <div class="d-flex justify-c-space-between">
+                        <p>Price (2 items)</p> <p>₹4,000</p>
```
Comment (shubhamkumar648) [inline pages/cart.html]: suggestion: you can also use a small or span element for small text! p tag is also good

**Review by shubhamkumar648 — APPROVED** [review]
well design, great use of the component library and utils variable, code looks good to me you can merge

File: pages/cart.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cart | CrickStock</title>
+    <link rel="shortcut icon" href="../assets/favicon.jpeg" type="image/x-icon">
+    <link rel="stylesheet" href="https://tarang-ui.netlify.app/stylesheets/main.css">
+    <link rel="stylesheet" href="../stylesheets/base.css">
+    <link rel="stylesheet" href="../styesheets/base.css">
```
Comment (krishanu2911) [inline pages/cart.html:10]: I think there is a typo.

**Review by krishanu2911 — APPROVED** [review]
Nice well-written code Everything seems good to go.

File: pages/cart.html (hunk 1 above)
Comment (kjais1720) [reply pages/cart.html]: Thank you for pointing it out, I have changed it in the latest commit

File: pages/cart.html (hunk 2 above)
Comment (kjais1720) [reply pages/cart.html:10]: Thanks for pointing it out, fixed it in the latest commit

**Pull request merged**